Core/Core.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 // Start of Core v.5.3.6-13ubuntu3.2
+use JetBrains\PhpStorm\ArrayShape;
```

Comment:
👀  where does that come from 😅

Reply:
https://github.com/JetBrains/phpstorm-stubs/blob/master/meta/attributes/ArrayShape.php